src/nidcpower/system_tests/test_system_nidcpower.py

```diff
@@ -214,134 +245,149 @@ def test_measure_multiple(session):
+@pytest.mark.channels('0')
+def test_create_and_delete_advanced_sequence(session):
     properties_used = ['output_function', 'voltage_level']
     sequence_name = 'my_sequence'
-    single_channel_session.source_mode = nidcpower.SourceMode.SEQUENCE
-    single_channel_session.create_advanced_sequence(sequence_name=sequence_name, property_names=properties_used, set_as_active_sequence=True)
-    single_channel_session.create_advanced_sequence_step(set_as_active_step=True)
-    assert single_channel_session.active_advanced_sequence == sequence_name
-    single_channel_session.output_function = nidcpower.OutputFunction.DC_VOLTAGE
-    single_channel_session.voltage_level = 1
-    single_channel_session.delete_advanced_sequence(sequence_name=sequence_name)
+    session.source_mode = nidcpower.SourceMode.SEQUENCE
+    session.create_advanced_sequence(sequence_name=sequence_name, property_names=properties_used, set_as_active_sequence=True)
+    session.create_advanced_sequence_step(set_as_active_step=True)
+    assert session.active_advanced_sequence == sequence_name
+    session.output_function = nidcpower.OutputFunction.DC_VOLTAGE
+    session.voltage_level = 1
+    session.delete_advanced_sequence(sequence_name=sequence_name)
     try:
-        single_channel_session.active_advanced_sequence = sequence_name
+        session.active_advanced_sequence = sequence_name
         assert False
     except nidcpower.errors.DriverError:
         pass
 
 
-def test_create_and_delete_advanced_sequence_bad_name(single_channel_session):
+@pytest.mark.channels('0')
+def test_create_and_delete_advanced_sequence_bad_name(session):
     properties_used = ['output_function_bad', 'voltage_level']
     sequence_name = 'my_sequence'
-    single_channel_session.source_mode = nidcpower.SourceMode.SEQUENCE
+    session.source_mode = nidcpower.SourceMode.SEQUENCE
     try:
-        single_channel_session.create_advanced_sequence(sequence_name=sequence_name, property_names=properties_used, set_as_active_sequence=True)
+        session.create_advanced_sequence(sequence_name=sequence_name, property_names=properties_used, set_as_active_sequence=True)
         assert False
     except KeyError:
         pass
 
 
-def test_create_and_delete_advanced_sequence_bad_type(single_channel_session):
+@pytest.mark.channels('0')
+def test_create_and_delete_advanced_sequence_bad_type(session):
     properties_used = ['unlock', 'voltage_level']
     sequence_name = 'my_sequence'
-    single_channel_session.source_mode = nidcpower.SourceMode.SEQUENCE
+    session.source_mode = nidcpower.SourceMode.SEQUENCE
     try:
-        single_channel_session.create_advanced_sequence(sequence_name=sequence_name, property_names=properties_used, set_as_active_sequence=True)
+        session.create_advanced_sequence(sequence_name=sequence_name, property_names=properties_used, set_as_active_sequence=True)
         assert False
     except TypeError:
         pass
 
 
```

Comment:
(Ignore the the line I'm writing this comment for)

For tests that result in the DeprecationWarning, please check if there's a way to suppress the warning from reaching the system test log.

Reply:
79c92402b3006af63f64e5433a682542bbb13e31